ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11.yaml

```diff
@@ -78,6 +78,19 @@ tests:
     - chain: cucushift-installer-rehearse-vsphere-ipi-provision
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-vsphere-ipi-disconnected
+  interval: 24h
+  steps:
+    cluster_profile: vsphere-discon
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @disconnected'
```

Comment:
@liangxia Looking at the build artifacts here https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/27401/rehearse-27401-periodic-ci-openshift-verification-tests-master-nightly-4.11-e2e-vsphere-ipi-disconnected/1509046010603638784/artifacts/e2e-vsphere-ipi-disconnected/cucushift-e2e/build-log.txt, it appears no test case is selectred.

Reply:
@jhou1 Fix in https://github.com/openshift/release/pull/27404

Reply:
Thanks!